src/lucky_record/associations.cr

```diff
@@ -20,6 +25,8 @@ module LuckyRecord::Associations
 
     field {{ assoc_name.id }}_id : Int32{% if nilable %}?{% end %}
 
+    association {{ model.resolve.constant(:TABLE_NAME).id }}, :id
```

Comment:
Could you use named args for these? `association table_name: {{ }}, foreign_key: :id` so it's a bit more clear what is going on

Reply:
Sure thing, that's much cleaner